<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReverbUI</class>
 <widget class="QWidget" name="ReverbUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReverbUI</string>
  </property>
  <widget class="QCheckBox" name="m_Enable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DelayTime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>DecayLevel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>FeedBack</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSpinBox" name="m_DelayTime">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="singleStep">
         <number>25</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="m_DecayLevel">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.005000000000000</double>
        </property>
        <property name="value">
         <double>0.300000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="m_FeedBack">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
